frontend/src/components/UploadArea/UploadArea.tsx

```diff
@@ -28,7 +30,12 @@ const UploadArea: FunctionComponent<UploadingAreaProps> = ({ value, onChange }):
         targets,
       });
     } catch (er) {
-      setParsedFile({ state: "failed-to-parse", reason: er.message });
+      setParsedFile(undefined);
+      notificationsUpdater({
+        type: "Add",
+        notificationType: "Error",
+        element: () => <span>{`Failed to parse file: ${er.response.data}`}</span>,
```

Comment:
If I upload an invalid file, I will get an error notification (which is expected and good) and the file will still be present in the upload area. I think it's better to reset it, so please do if it's not hard.

![image](https://user-images.githubusercontent.com/2556409/117403053-3ba5ef00-af10-11eb-8560-8dfe45df95d4.png)